generate/convert.go

```diff
@@ -262,6 +263,13 @@ func (g *generator) convertType(
 		// options work, recursing here isn't as connvenient.)
 		// Note this does []*T or [][]*T, not e.g. *[][]T.  See #16.
 		goTyp = &goPointerType{goTyp}
+	} else if !typ.NonNull && g.Config.Optional == "generic" {
+		qualifiedTypeName := regexp.MustCompile("[^/]+$").FindString(g.Config.OptionalGenericType)
+
```

Comment:
I saw that you already have functionality for dealing with fully qualified to just qualified import names inside the `(*generator).ref` method, but it's mixed in with logic around registering imports and so didn't feel appropriate to use here.

Reply:
Yeah that's true, I could easily just register the type here. Am I correct in understanding that the return value of `(*generator).ref` is the `packagename.Type` string?